### PR TITLE
docs: update imprint and contact

### PR DESCRIPTION
**Pull request by lmoesle**
### Description

Short description or comments

### Reference

Issues: closes #xxx

### Check-List

- [ ] All Acceptance criteria of user story are met
- [ ] JUnit tests are written (60% CodeCov)
- [ ] [Internal Review]([https://github.com/it-at-m/digiwf-core/blob/dev/CHANGELOG.md](https://confluence.muenchen.de/display/MPdZ/Review+-+DigiWF)) is maintained
- [ ] Documentations [external](https://github.com/it-at-m/digiwf-core/tree/dev/docs) and [internal](https://wiki.muenchen.de/betriebshandbuch/index.php?title=DigiWF&sfr=betriebshandbuch) are completed
- [ ] Smoketest successful (Manual E2E-Test depending on Change) 
- [ ] No Branch waste left
- [ ] [Board](https://app.zenhub.com/workspaces/digiwf-621f70bf50ea1100120b7e93/board) is up-to-date
- [ ] Openshift environments are prepared (Secrets, etc.) and release-issue is maintained

### CODE REVIEW

**Comment by lmoesle** [comment]
I copy & pasted the impress from opensource.muenchen.de

File: docs/src/imprint/README.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+
+Sie haben als betroffene Person, das Recht auf Auskunft, das Recht auf Berichtigung oder Löschung, das Recht auf Einschränkung der Verarbeitung und das Recht auf Widerspruch gegen die Verarbeitung deiner Daten.
+
+Bitte richten sie ihren Widerspruch formlos an oben genannte Adresse.
+
+Darüber hinaus haben sie das Recht auf Datenübertragbarkeit. Sie haben weiter das Recht, dich bei einer Aufsichtsbehörde über die Verarbeitung zu beschweren. Eine Liste der entsprechenden Behörden finden sie unter: https://www.bfdi.bund.de/DE/Infothek/Anschriften_Links/anschriften_links-node.html.
+
+### Hosting
+
+Der Inhalt dieser Seite wird auf [Github verwaltet](https://github.com/it-at-m/opensource.muenchen.de), dort als Containerimage erstellt und dann im Rechenzentrum der Landeshauptstadt München gehostet.
```
Comment (darenegade) [inline docs/src/imprint/README.md]: Hier ist noch anpassen

**Review by darenegade — CHANGES_REQUESTED** [review]
Nur das eine sonst, passt es

File: docs/src/imprint/README.md (hunk 1 above)
Comment (lmoesle) [reply docs/src/imprint/README.md]: Ich hab's angepasst

**Review by darenegade — APPROVED** [review]

**Pull request merged**